Add prompt option to blogging reminders settings

wargcm opened the pull request:
See: #18316

## Description

Makes the following changes:

- Adds the UI for the blogging prompts title, description, help button,
  and switch to turn the feature on/off
- Removes the spacer view after the next button and uses a constant
  value for spacing on iPad
- Adds tracking to the prompt help button press and to the switch that
  turns the feature on/off

## Testing

To test:

- Enable blogging prompts feature flag
- Select a site
- Go to Configure > Site Settings
- Open Blogging Reminders
- Validate UI is correctly laid out
- Tap the help button
  - Verify screen properly navigates to the blogging prompts info screen
  - Verify help button fires tracking on tap
- Go back to setting screen
- Tap switch and verify tracking on on/off

## Screenshots

| Before | After |
|-------|-------|
| ![Simulator Screen Shot - iPhone 11 - 2022-04-19 at 11 46 33](https://user-images.githubusercontent.com/2454408/164043771-62817e47-ee5b-4fcf-ab12-bcc8760bca09.png) | ![Simulator Screen Shot - iPhone 11 - 2022-04-19 at 11 44 53](https://user-images.githubusercontent.com/2454408/164043838-87053b82-12b4-4dd2-8600-36dca653ea1b.png) |
| ![Simulator Screen Shot - iPhone 11 - 2022-04-19 at 11 46 39](https://user-images.githubusercontent.com/2454408/164043968-7466cdb6-5e72-4772-9437-5cf911441ba5.png) | ![Simulator Screen Shot - iPhone 11 - 2022-04-19 at 11 45 15](https://user-images.githubusercontent.com/2454408/164044030-a84cb0e9-98bf-435f-b18e-7e006a044bc7.png) |

## Regression Notes

1. Potential unintended areas of impact
Blogging reminders settings layout

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Validated the screen in dark mode, different text sizes, and on iPad

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

wargcm committed:
Add prompt option to blogging reminders settings

Makes the following changes:

- Adds the UI for the blogging prompts title, description, help button,
  and switch to turn the feature on/off
- Removes the spacer view after the next button and uses a constant
  value for spacing on iPad
- Adds tracking to the prompt help button press and to the switch that
  turns the feature on/off

See: #18316

ScoutHarris commented:
With large text, the `bloggingPromptsTitle` gets cut off. This is Accessibility XL:

<img width="450" alt="large_text" src="https://user-images.githubusercontent.com/1816888/164101305-1eb7c325-246b-4fd4-9e85-6068c917b540.png">

I know space is limited in this view, but it looks like there is enough room for it to expand before it hits the action button:

<img width="432" alt="space" src="https://user-images.githubusercontent.com/1816888/164101946-b85302e9-6a94-4a6f-966b-a1a45167273e.png">

Since this is a disabled feature, if you'd prefer to fix it separately, that's acceptable. Or you can fix it on this PR. Either way is fine. Just let me know and I'll review this PR accordingly.

ScoutHarris reviewed (changes requested):
See comment about large text.

wargcm committed:
Fix prompt text when using large accessibility text

wargcm commented:
Thanks for the catch! I ended up making the following changes:

- Removed a couple stack views and used a regular view instead to get it to layout properly
- Changed the spacing after the prompt view and spacer view to zero so it properly expands into that space

![Simulator Screen Shot - iPhone 11 - 2022-04-20 at 12 27 45](https://user-images.githubusercontent.com/2454408/164286389-59173fdd-a311-4865-be41-7a100a3482d7.png)

There are still some issues with the other labels on this screen with large accessibility text. I think those will be resolved though when this view moves to a scroll view.

ScoutHarris reviewed (approved):
Oh yea that's much better! 👍 

> There are still some issues with the other labels on this screen with large accessibility text. I think those will be resolved though when this view moves to a scroll view.

Agreed.

:shipit:
